CHANGELOG.md

```diff
@@ -8,38 +8,13 @@ Commits have to follow following convention: https://www.conventionalcommits.org
 
```

Comment:
Should not be touched

Reply:
That the thing I don't touch. It comes from the occuring merged from eclispe/kiso to my fork when I synchronize them.